x-pack/plugins/security_solution/server/endpoint/lib/artifacts/saved_object_mappings.ts

```diff
@@ -34,3 +34,31 @@ export const manifestType: SavedObjectsType = {
   mappings: manifestSavedObjectMappings,
   migrations,
 };
+
+export const unifiedManifestSavedObjectType = ManifestConstants.UNIFIED_SAVED_OBJECT_TYPE;
+
+export const unifiedManifestSavedObjectMappings: SavedObjectsType['mappings'] = {
+  dynamic: false,
+  properties: {
+    artifactIds: {
+      type: 'text',
```

Comment:
should this be `type: "array"` instead?

Reply:
No `array` type in ES, every field can be a value or an array of values as long as they all share the same type.
https://www.elastic.co/guide/en/elasticsearch/reference/current/array.html

Reply:
and what about using `keyword` both here and for `semanticVersion`?

Elasticsearch [analyses/tokenizes/normalizes](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-overview.html) `text` types, so they can be more naturally searched, which is not needed for these types, so we can spare the overhead for text analysis

Reply:
Great stuff, I had wrong idea of what keyword implies, thanks @gergoabraham! 
eda8443785d1900eea14540dbf7f0846c0315e36